src/Microsoft.DotNet.GitHub.IssueLabeler/Controllers/PredictionEngineController.cs

```diff
@@ -14,19 +14,19 @@
 
 namespace Microsoft.DotNet.GitHub.IssueLabeler
 {
-    [Route("api/WebhookIssue")]
-    public class WebhookIssueController : Controller
+    [Route("api/PredictionEngine")]
```

Comment:
This is a breaking change that will break Hubbup. Maybe we can just _add_ the new route and have both enabled until I can migrate Hubbup to use the new URL?

Reply:
Created https://github.com/dotnet/issue-labeler/pull/37